test/lint/lint-assertions.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018-2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Check for assertions with obvious side effects.
+
+import sys
+import subprocess
+
+
+def exec_cmd(command: [], error_msg: ""):
+    try:
+        output = subprocess.check_output(command, universal_newlines=True, encoding="utf8")
```

Comment:
You should check the output for being non-empty, like in the original script.

Reply:
If it prints the output, so it isn't empty.